src/main/java/frc/robot/Robot.java

```diff
@@ -18,14 +19,17 @@ public class Robot extends TimedRobot {
 
   private final RobotContainer m_robotContainer;
 
+  private MatchStateUtil matchState = new MatchStateUtil(false, true, false);
+
+
   /**
    * This function is run when the robot is first started up and should be used for any
    * initialization code.
    */
   public Robot() {
```

Comment:
could be mistaken but don't think there is a Robot constructor. last year these were just initialized in robotInit()